argus/src/main/java/com/moldedbits/argus/provider/login/EmailLoginProvider.java

```diff
@@ -26,13 +23,14 @@
 /**
  * Allow user to login with email and password
  */
-public abstract class EmailLoginProvider extends BaseProvider {
+public abstract class EmailLoginProvider extends BaseProviderHandler {
```

Comment:
Use of inheritance doesn't seem right for applying a theme. `BaseProviderHandler` is a vague name.